src/integrations.py

```diff
@@ -113,9 +113,12 @@ def load_bind_account_from_remote_ldap(self) -> None:
         )
 
     @property
-    def ldap_url(self) -> str:
-        hostname = self._charm.config.get("hostname") or socket.getfqdn()
-        return f"ldap://{hostname}:{GLAUTH_LDAP_PORT}"
+    def ldap_url(self) -> List[str]:
```

Comment:
Just a minor nitpick on naming: probably then we could rename it to `ldap_urls`?

(I could be wrong) Another minor thing is about the reason that `ldap_url` was changed to a container type initially. @nsklikas has more authority to clarify that. I remember it's somehow related to glauth proxy usage.

What I worry here is that this data container will represent inconsistent meanings when integrating with traefik or not (in a proxy mode).

Let me know if I am wrong :)

Reply:
Yes, it was related to https://github.com/canonical/ldap-integrator/. 

The reason is that glauth accepts a list of URLs when configuring it to acts as an ldap proxy (see https://glauth.github.io/docs/backends.html). When making an ldap request to the external server, tries to call them one by one, until one the requests succeeds.

I agree with the name change

Reply:
Renaming the property to `ldap_urls` sounds good to me. Getting a list of possible endpoints for the LDAP server is also ideal for SSSD as you can configure to switch server endpoints if one endpoint fails: https://manpages.ubuntu.com/manpages/oracular/en/man5/sssd-ldap.5.html#failover.

Glauth high-availability is also achieved by having multiple running instances of Glauth functioning as their own individual endpoint, but share the same database backend.